src/pages/spawn/spawnVolume/spawnVolumeTable/spawnVolumeTableActions/UnmountBtn.tsx

```diff
@@ -33,27 +35,40 @@ export const UnmountBtn: React.FC<Props> = ({ volume }) => {
     : volume.host?.id;
 
   return (
-    <Popconfirm
-      icon={null}
-      placement="left"
-      title={`Detach this volume ${volumeName} from host ${hostName}?`}
-      onConfirm={() => {
-        spawnAnalytics.sendEvent({
-          name: "Unmount volume",
-          volumeId: volume.id,
-        });
-        detachVolume({ variables: { volumeId: volume.id } });
-      }}
-      okText="Yes"
-      cancelText="Cancel"
+    <ConditionalWrapper
+      condition={volume.homeVolume}
```

Comment:
volume.homeVolume tells us that at a volume is set up to serve as a home volume for a host. However, it is possible to utilize a volume that was initialized as a home volume as a non-home volume. I think this condition should actually be based on this boolean expression which answers "Is this volume currently serving as a home volume for a host?" `myHosts.reduce((accum, h) => accum || h.homeVolume === volume.Id, false)`